src/istgt_lu_ctl.c

```diff
@@ -2959,17 +2960,31 @@ istgt_uctl_cmd_iostats(UCTL_Ptr uctl)
 		size = malloc(length + 1);
 		snprintf(size, length + 1, "%"PRIu64, spec->size);
 
+        usedlogicalblock = ( spec->stats.used / spec->blocklen ) ;
+
+        length = snprintf(NULL, 0, "%"PRIu64, usedlogicalblock);
+		usedblocks = malloc(length + 1);
+		snprintf(usedblocks, length + 1, "%"PRIu64, usedlogicalblock);
+
+        length = snprintf(NULL, 0, "%"PRIu64, spec->blocklen);
+		sectorsize = malloc(length + 1);
+		snprintf(sectorsize, length + 1, "%"PRIu64, spec->blocklen);
+
 		json_object *jreads = json_object_new_string(reads);	/* instantiate child object */
 		json_object *jwrites = json_object_new_string(writes);
 		json_object *jtotalreadbytes = json_object_new_string(totalreadbytes);
 		json_object *jtotalwritebytes = json_object_new_string(totalwritebytes);
 		json_object *jsize = json_object_new_string(size);
+		json_object *jusedlogicalblocks = json_object_new_string(usedblocks);
```

Comment:
don't we need to free/delete json objects at the end?

Reply:
already freeing at the end

Reply:
perfect.